src/components/Leaderboard/Leaderboard.jsx

```diff
@@ -0,0 +1,70 @@
+import axios from 'axios';
```

Comment:
Wasn't sure where to put this but - 
I think we should have a way of indicating who is #1 in the leaderboard beyond them just being at the top. Maybe we could give the top 3 players medals next to their names, or at the least, a number next to everyones names so they can know what 'place' they are in (eg 10th)

Reply:
Alternatively, if our leaderboard only wants to show the top X amount of players, I think that could work too!